src/jobBundle/Controller/JobController.php

```diff
@@ -16,7 +16,6 @@
 use Symfony\Component\Form\Extension\Core\Type\SubmitType;
 use Symfony\Component\Form\Extension\Core\Type\TextType;
 
-
 class JobController extends Controller
```

Comment:
Controllers are for controlling the flow:
- accepting the requests
- directing flow to desired services
- render the response

Please extract all other functionality to other layers (e.g.: service).